src/Sparrow/Json/BlittableJsonTextWriter.cs

```diff
@@ -170,16 +169,63 @@ public unsafe void WriteString(LazyStringValue str)
             if (numberOfEscapeSequences == 0)
             {
                 WriteRawString(strBuffer, size);
+            }
+            else
+            {
+                UnlikelyWriteEscapeSequences(str, numberOfEscapeSequences, escapeSequencePos, strBuffer, size);
+            }
 
-                EnsureBuffer(1);
-                _buffer[_pos++] = Quote;
+            EnsureBuffer(1);
+            _buffer[_pos++] = Quote;
+        }
+
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public void WriteStringJoin(IEnumerable<LazyStringValue> strArray)
```

Comment:
No, why would we ever want to do something like that? You can't undo this to component parts easily.
Why not use an array?